docs/docs/segment-path.md

```diff
@@ -34,7 +34,7 @@ Display the current path.
 - windows_registry_icon: `string` - the icon to display when in the Windows registry - defaults to `\uE0B1`
 - style: `enum` - how to display the current path
 - mapped_locations: `map[string]string` - custom glyph/text for specific paths(only when `style` is set to `agnoster`,
-`agnoster_full` or `short`)
+`agnoster_full`, `agnoster_short`, `short`, or `folder`)
```

Comment:
`agnoster_short` already supported `mapped_locations`, it just wasn't mentioned in the docs.